.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing
+
+Thank you for your interest in contributing to the venia-pwa-concept project! Before you start contributing, please take a moment to read through the following guidelines:
+
+* [Code of Conduct]
+* [Support]
+
+To contribute to this repository, start by forking the [official repository] and following the installation instructions in the README file.
+
+## Pull Request checklist
+
+* Code contributions must adhere to the [Coding style guide](#coding-style-guide)
+* PR commits should contain [meaningful commit messages]
+* To help with reviews, your PR should only create/revise a single feature or fix a single issue.
+* If your PR fixes a bug, please provide a step-by-step description of how to reproduce the bug.
+* If your PR addresses an existing issue, please reference that issue in the title or description.
+
+### Coding style guide
+
+**TODO: Write style guide**
+
+## Contribution process
+
+**TODO: Write contribution process**
```

Comment:
Here's my quick draft of the contribution process we discussed this week.

## Contribution process

Magento maintains the roadmap for this and other `magento-research` repositories publicly, in GitHub Issues. Community participation in this backlog is encouraged and appreciated. Even foundational infrastructure stories are available for a generous developer to take on. Issues tagged with the **"help wanted"** label are prepared for community ownership. *We will also gratefully accept unsolicited new issues and pull requests, but we give a priority to issues from our roadmap that community developers are kind enough to take on.*

### Claiming a Roadmap Issue

If you want to contribute by taking on a **"help wanted"** issue from, we ask that you go through a simple, short process with us so we can forecast the progress of our projects.

1. To offer to take the issue, simply add a comment on the issue expressing your interest! Make sure that your GitHub profile includes an email address where we can contact you privately.
1. A maintainer will contact you and ask to set up a real-time meeting about the issue. This meeting can be in person, on video chat, on audio chat, or text-only, as long as it's close to real time. The meeting will be brief, but may vary with the complexity of the issue.
1. During this chat, we will provide you with any additional materials or resources you need. Occasionally, we may schedule follow-up meetings for a larger issue.
1. Finally, **we will ask you to provide an estimated date of completion.** You can give this estimate at the end of the meeting, or if the scoping requires more time, you can take up to one week (five business days) to provide an estimated date of completion.
  - If you can't get back to us by the end of that week, we understand! Community developers are helping out in addition to your regular jobs. We will un-assign you from this issue, but please feel free to contribute on another.
  - Once you provide us with that estimated date, we'll consider the issue to be "in progress" and **we'll make you a member of the `magento-research` organization, with access to private discussions and repositories.**
1. If you contact us to say it will take longer than your estimated time, please do so before the estimated date of completion. We may request an update on your progress, but otherwise it should be all right to reschedule.
1. If your estimated date of completion passes and we haven't heard from you, we'll wait one week, then un-assign you from the issue and put it back in the queue.
1. When you're finished with the issue, create a pull request mentioning the issue by number in the pull request title or body. We'll start the (brief) code review process. Once we have accepted and merged your contribution, you'll become an official contributor! Official contributors are invited to backlog grooming sessions and have direct influence over the product roadmap.

We hope this short guide helps to paint a clear picture of your role in the contribution process. Thank you in advance for helping with our research projects!